### PR TITLE
Webhook support for api_version and private_metafield_namespaces

### PR DESCRIPTION
**Pull request by sshaw**

### CODE REVIEW

File: webhook.go [hunk 1]
```diff
@@ -27,14 +27,16 @@ type WebhookServiceOp struct {
...
-	Fields              []string   `json:"fields"`
-	MetafieldNamespaces []string   `json:"metafield_namespaces"`
+	ID                         int64      `json:"id"`
+	Address                    string     `json:"address"`
+	Topic                      string     `json:"topic"`
+	Format                     string     `json:"format"`
+	CreatedAt                  *time.Time `json:"created_at,omitempty"`
+	UpdatedAt                  *time.Time `json:"updated_at,omitempty"`
+	Fields                     []string   `json:"fields"`
+	MetafieldNamespaces        []string   `json:"metafield_namespaces"`
```
Comment (sshaw) [inline webhook.go:37]: I only added lines 38-39 below. The rest is from `go fmt`.

**Comment by oliver006** [comment]
Can you rebase your change please and resolve the merge conflicts? Thanks!

**Comment by sshaw** [comment]
@oliver006 updated!

File: webhook_test.go [hunk 2]
```diff
@@ -33,7 +33,17 @@ func webhookTests(t *testing.T, webhook Webhook) {
...
-		t.Errorf("Webhook.Fields returned %+v, expected %+v", webhook.MetafieldNamespaces, expectedArr)
+		t.Errorf("Webhook.MetafieldNamespaces returned %+v, expected %+v", webhook.MetafieldNamespaces, expectedArr)
+	}
+
+	expectedArr = []string{"info-for", "my-app"}
+	if !reflect.DeepEqual(webhook.PrivateMetafieldNamespaces, expectedArr) {
+		t.Errorf("Webhook.PrivateMetafieldNamespaces returned %+v, expected %+v", webhook.PrivateMetafieldNamespaces, expectedArr)
+	}
+
+	expectedStr = "2021-01"
```
Comment (oliver006) [inline webhook_test.go:44]: I think we have this test twice now.

**Review by oliver006 — COMMENTED** [review]
Thanks for the quick response!
One minor issue, then this should be good to go.

File: webhook_test.go (hunk 2 above)
Comment (oliver006) [reply webhook_test.go:44]: 🎉

**Review by oliver006 — APPROVED** [review]
🚀

**Pull request merged**